docs/Contributing/architecture/system_design/UI.md

```diff
@@ -0,0 +1,56 @@
+[Back to top](./README.md)
+# Fleet UI's
+
+## Fleet Server
+Single-page web app (React / Typescript in `frontend/`) served by server under `/` path. Auth flows:
+* Session cookie after login/SSO
+* CSRF protection via cookie+header (if applicable)
+* Real-time refresh: polling; optionally websockets for live query results
+
+Key sections -> API domains:
+| UI Page | Backend Endpoints |
+|---------|-------------------|
+| Hosts table | `/api/latest/fleet/hosts` (filters, pagination) |
+| Host details | `/api/latest/fleet/hosts/{id}` (software, MDM, policies) |
+| Queries | `/queries`, `/queries/run` (campaign start) |
+| Policies | `/policies`, host policy status endpoints |
+| Software | `/software/titles`, `/software/{id}` |
+| MDM Profiles | `/mdm/apple/profiles`, `/mdm/windows/configurations` |
+| Scripts | `/scripts`, `/scripts/run` |
+
+```mermaid
+flowchart LR
+	Browser -->|HTTPS JSON| API[REST API]
+	Browser -->|Websocket (optional)| WS[Live Query Stream]
+	API --> DB[(MySQL)]
+	API --> Cache[(Redis)]
+```
```

Comment:
Should there be a section for the `fleetdm.com` sails web app? I plays important roles for some features (beyond being the company's website).